apps/sms/index.html

```diff
@@ -145,7 +145,7 @@ <h1 id="threads-edit-mode" data-l10n-id="selectThreads-title"></h1>
           <div class="bottom-tab" role="region" >
             <ul class="bb-tablist">
               <li role="presentation">
-                <button id="threads-delete-button" data-action="delete-button"></button>
+                <button id="threads-delete-button" class="disable-when-dialog-animates" data-action="delete-button"></button>
```

Comment:
Can we use a ```disabled``` attribute instead? I feel that it's more semantic and clear in meaning.